backend/app/services/shared_task.py

```
@@ -0,0 +1,630 @@
+            try:
+                user_id = int(user_id_str)
+                task_id = int(task_id_str)
+            except ValueError:
+                return None
+
+            # If database session is provided, query user_name and task_title from database
+            if db is not None:
+                # Query user name
+                user = (
+                    db.query(User)
+                    .filter(User.id == user_id, User.is_active == True)
```

Ruff (0.14.7)

Avoid equality comparisons to `True`; use `User.is_active:` for truth checks

Replace with `User.is_active`

(E712)